### ISSUE TITLE
BigDecimal.new is deprecated warnings

### ISSUE THREAD

**Issue by starzonmyarmz** [bigcartel/dugway#170]
I recently updated the Dugway gem to `0.12.0` and now when running `dugway server` I get the following warnings in my Terminal:

```
/Library/Ruby/Gems/2.6.0/gems/dugway-0.12.0/lib/dugway/liquid/filters/util_filters.rb:95: warning: BigDecimal.new is deprecated; use BigDecimal() method instead.
```

Thoughts? Is this an isolated issue, or something you're already aware of?

**Comment by brettchalupa** [comment]
Thanks for the heads up @starzonmyarmz! I believe this has to do a new deprecation when using Ruby 2.6.0. I've got a fix brewing in https://github.com/bigcartel/dugway/pull/171. I'll drop a note here when I close the issue when a new gem is cut and pushed to rubygems.org.

**Comment by starzonmyarmz** [comment]
Awesome! Thanks Brett!

**Comment by brettchalupa** [comment]
@starzonmyarmz fix for this up in the latest version of the gem. `gem install dugway` will do the trick. The fix is in version 0.12.1. `dugway version` will let you know what you've got installed locally. Let us know if you run into any issues!

**Comment by starzonmyarmz** [comment]
Works great! Thanks again, @brettchalupa!
